data/devices/logitech-g303-shroud-wireless.device

```diff
@@ -0,0 +1,8 @@
+[Device]
+Name=Logitech G303 Shroud Wireless
+DeviceMatch=usb:046d:c547;usb:046d:c097
```

Comment:
```suggestion
DeviceMatch=usb:046d:c097
```

Reply:
I was hoping I could just apply this and merge but ... apparently I can't. Please apply this locally, force-push and we can getr this merged. Thanks

Reply:
Sorry I missed this! Just to clarify, in my testing that did NOT work.

Reply:
weird. we can't merge the c547 ID though because that one is shared across many devices. the CI *should* have picked this up, we have a test specifically for that, not sure why it didn't pick up on it.

Either way, the kernel is supposed to handle this and expose the device itself with the correct PID - apparently that doesn't work for these ones (yet? are you on a recent-enough kernel)?